titus-client/src/main/java/com/netflix/titus/runtime/endpoint/v3/grpc/GrpcJobManagementModelConverters.java

```diff
@@ -362,7 +362,11 @@ public static Container toCoreContainer(com.netflix.titus.grpc.protogen.Containe
     }
 
     private static List<BasicContainer> toCoreBasicContainers(List<com.netflix.titus.grpc.protogen.BasicContainer> extraContainersList) {
-        return extraContainersList.stream().map(GrpcJobManagementModelConverters::toCoreBasicContainer).collect(Collectors.toList());
+        List<BasicContainer> basicContainers = Collections.emptyList();
```

Comment:
IIRC, this won't work as `emptyList` is a specific implementation that must be empty. You can change it to `new ArrayList()`

Reply:
Ah you are right, they are immutable